test/unit/vendor/lime/fixtures/pass_with_one_parse_error.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+require_once __DIR__.'/../../../../bootstrap/unit.php';
+
+$test = new lime_test();
+
+parse_error
```

Comment:
This file should be excluded from everywhere that parses php files. At this moment php-cs-fixer, but later phpstan, rector etc.

Reply:
php-cs-fixer exclude all files that contain "vendor" on its path.

This means that the directory "test/unit/vendor" is ignored.

Fixing that, will requires a structural correction, that is out of scope of this PR.

@connorhu What can I do on the scope of this PR?

Reply:
Indeed, you are right! For the other tools, it was just a reminder to myself.

Reply:
1. [configure php-cs-fixer](https://github.com/FriendsOfSymfony1/symfony1/pull/371/commits/4523808934ecc5841757b0d1c5dbbd684d54c16f)
2. [execute CS fixer](https://github.com/FriendsOfSymfony1/symfony1/pull/371/commits/54a52f8af7edfc7e41e0c590926b99e46f87e289)
3. [Allow having pretty method name for test cases](https://github.com/FriendsOfSymfony1/symfony1/pull/371/commits/7dccdd1e09cd02d5e6e3afbb7219f596b798bbfb)